packages/lending/src/protocol.utils.ts

```diff
@@ -125,3 +125,27 @@ export const removePortfolioDynamicFields = (portfolio: any) => {
     });
   }
 };
+
+export const getLstApyFromMap = (address: string, lstTokenAPYMap: Record<string, string>) => {
+  return lstTokenAPYMap[address.toLowerCase()] || '0';
+};
+
+export const calcSupplyGrossApy = (apy: string, lstApy: string) => {
+  if (lstApy === '0') {
+    return apy;
+  } else if (apy === '0') {
+    return lstApy;
+  } else {
+    return BigNumberJS(apy).plus(lstApy).toString();
```

Comment:
suggest using
`return lstApy === '0' ? apy : apy === '0' ? lstApy : BigNumberJS(apy).plus(lstApy).toString();`
to be cleaner

Reply:
changed